app/league/dashboard/views.py

```diff
@@ -173,7 +173,7 @@ def create_game():
             episode=form.episode.data,
             played_at=played_at
         )
-        messenger.notify_slack({'text':repr(game)})        
+        messenger.notify_slack(repr(game))
```

Comment:
This still isn't really what I'd call "human friendly". `repr` just gets us a string (roughly) indicates how the object is constructed. What we really want is a message like "Andrew Hall (W) defeated Jason McGibbon (B) with 2 stones, 0.5 komi at so and so date and time"